src/components/Map/Map.js

```diff
@@ -3,9 +3,25 @@ import mapboxgl from 'mapbox-gl';
 import './Map.css';
 import { statesData } from './us-states.js';
 import { MAPBOX_TOKEN } from './tokens.js';
+import { layerColors } from './layerColors.js';
 
 mapboxgl.accessToken = MAPBOX_TOKEN;
 
+export const enumDataLayerType = {
+  ACTIVES: {
```

Comment:
If I'm understand this part correctly, you are making just three predetermined layers, correct? I believe in the future Peter wants it to be more flexible based on headings, so would it be possible later on to allow the number of layers to be dynamic?

Reply:
Yes definitely! So if we need to generate dynamic layers in the future, we'll dynamically make some json like this (since enum is basically just json in JS).

Reply:
Okay awesome! Thanks!